src/jit/importer.cpp

```diff
@@ -20091,29 +20271,50 @@ void Compiler::impDevirtualizeCall(GenTreeCall*            call,
     }
 #endif // defined(DEBUG)
 
-    if (!isExact && !objClassIsFinal && !derivedMethodIsFinal)
+    bool canDevirtualize = true;
```

Comment:
This could also be written as:

`canDevirtualize = isExact || objClassIsFinal || (isInterface && derivedMethodIsFinal);`